pages/documentation/migration-V3.md

```diff
@@ -0,0 +1,822 @@
+      <li> 
+        Search for <strong>includePaths</strong> in your project's Gulp files. The paths in this list are where the Sass compiler looks for your source files. In USWDS 3.0
+        <p><strong>Old</strong></p>
+        <pre><code>.pipe(
+  sass({
+    includePaths: [
+      PROJECT_SASS_SRC,
+      `${USWDS}/scss`,
+      `${USWDS}/scss/packages`,
+    ],
+  })
+)</code></pre>
+        <p><strong>New</strong></p>
+        <pre><code>.pipe(
+  sass({
+    includePaths: [
+      PROJECT_SASS_SRC,
+      `${USWDS},
+      `${USWDS}/packages`,
+    ],
+  })
+)</code></pre>
+      </li>
+      <li>Recompile your Sass as usual. When it compiles, it is now using USWDS 3.0!</li>
+    </ol>
+  </div>
+
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a7"
+    >
+      If you're using USWDS Compile
+    </button>
+  </h4>
+  <div id="m-a7" class="usa-accordion__content usa-prose"  markdown="1">
+```
+/* gulpfile.js */
+
+const uswds = require("@uswds/compile");
+
+/**
+ * USWDS version
+ */
+
+uswds.settings.version = 3;
+```
+  </div>
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a8"
+    >
+      If you're using a custom Gulp workflow
+    </button>
+  </h4>
+  <div id="m-a8" class="usa-accordion__content usa-prose">
+    TK   
+  </div>
+
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a9"
+    >
+      If you're using Webpack
+    </button>
+  </h4>
+  <div id="m-a9" class="usa-accordion__content usa-prose">
+    TK   
+  </div>
+</div>
+
+### 5. Update to Sass module syntax
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+USWDS 3.0 supports the deprecated Sass import syntax. You don't need to change to the new syntax in your project Sass to use USWDS 3.0. But once you update, we recommend updating to the new syntax as well.  You’ll need to both switch @import to @use, and then create @use project-specific settings.  In USWDS 3.0 with Sass modules, instead of having multiple theme files that contain every setting available to the design system, you only keep track of the settings you changed in a single file. 
+
+We recommend creating a new file to keep track of these changed settings. These instructions will help you create that file, format it properly, and use it in your project. 
+
+You can complete these changes by taking the following steps: 
+
+1. Replace all instances of **@import** with **@forward**, using find/replace in your Sass file.
+2. Locate your existing project theme files. These are the _uswds-theme files that you found in [Step 1: Check your current USWDS code and settings versions](#1-check-your-current-uswds-code-and-settings-versions). (If you already have your project-specific settings file in a single file, you can skip ahead Step 4. If not, you'll need to collect your project-specific settings in a single file.)
```

Comment:
@bonnieAcameron Where is step 4? Are we missing steps here?

Reply:
Oh I think we mean step 4 in the entire installation process. Good catch! Can we add an "anchor" link to "step 4" to add clarity?

Reply:
@bonnieAcameron just realized that this is nested within step 5. Should I update the text to say skip to step 6?

Reply:
@mahoneycm , how can I see the most recent mockup of the page? I'm starting to feel a bit disoriented and I'd love to see it in action.

Reply:
Would it be helpful if we clarified how we label steps? We have the main steps 1-7, then any nested steps can switch to alpha (a., b., c...) or roman numbers (i., ii., iii...)? Then we could refer something like Step 3a?

Reply:
@bonnieAcameron At the top of this PR there's a [preview link!](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/cm-migrate-3.0/documentation/migration/)

![image](https://user-images.githubusercontent.com/25211650/164784654-6184b4c9-b343-4375-ad41-21909152b88b.png)

Reply:
@amyleadem that's a great idea!

Reply:
Where did we land with the nested steps?